requests/adapters.py

```diff
@@ -204,11 +198,13 @@ def proxy_manager_for(self, proxy, **proxy_kwargs):
 
         return manager
 
-    def _update_poolmanager_ssl_kw(self, verify, cert):
-        """Update the :class:`PoolManager <urllib3.poolmanager.PoolManager>`
-        connection_pool_kw with the necessary SSL configuration. This method
-        should not be called from user code, and is only exposed for use when
-        subclassing the :class:`HTTPAdapter <requests.adapters.HTTPAdapter>`.
+    @staticmethod
+    def _pool_kwargs(verify, cert):
```

Comment:
Don't have static methods: they're just not very useful. Either we should decide users can't override this, in which case this should be a function, or we should decide they can and it becomes an instance method.